.github/workflows/deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy Rovercode UI
+on:
+  push:
+    branches:
+    - alpha
+    - beta
+    - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env to alpha
+        if: endsWith(github.ref, '/alpha')
+        run: |
+          echo "FRONTEND_SENTRY_DSN=https://2877691aaae24810ab09890efef576be@sentry.rovercode.com/6" >> $GITHUB_ENV
+          echo "PXT_HEX_URL=https://rovercode-gigglebot-mbed.s3.us-east-2.amazonaws.com/alpha/rovercode.hex" >> $GITHUB_ENV
+          echo "SUBSCRIPTION_SERVICE=https://alpha-payment.rovercode.com" >> $GITHUB_ENV
+          echo "STRIPE_SHARABLE_KEY=pk_test_51HFVMDDAGjnnjW0cTIvpN2q1eigGLhpgdu3hI5qwWqfd5LPgDpuvyTCFOIiyV1ink662rRNAIPkvjD1FAf5SJFY400rlIgsZ4P" >> $GITHUB_ENV
+
+      - name: Set env to beta 
+        if: endsWith(github.ref, '/beta')
+        run: |
+          echo "FRONTEND_SENTRY_DSN=https://7b9bd38281074edca423de6c7eff2ba6@sentry.rovercode.com/7" >> $GITHUB_ENV
+          echo "PXT_HEX_URL=https://rovercode-gigglebot-mbed.s3.us-east-2.amazonaws.com/beta/rovercode.hex" >> $GITHUB_ENV
+          echo "SUBSCRIPTION_SERVICE=https://beta-payment.rovercode.com" >> $GITHUB_ENV
+          echo "STRIPE_SHARABLE_KEY=pk_test_51HFVMSGHtsId1yLpDXJ5454Ic0LON1vy2mEeD56VXZbEsoPs2DTW3swZWUn9HGn2xy8wLjOtnxdo4J6v2387cDFw00rWbfJrpA" >> $GITHUB_ENV
+
+      - name: Set env to production
+        if: endsWith(github.ref, '/master')
+        run: |
+          echo "FRONTEND_SENTRY_DSN=https://938efcaf982044529be14f28ea5afda2@sentry.rovercode.com/5" >> $GITHUB_ENV
+          echo "PXT_HEX_URL=https://rovercode-gigglebot-mbed.s3.us-east-2.amazonaws.com/prod/rovercode.hex" >> $GITHUB_ENV
+          echo "SUBSCRIPTION_SERVICE=https://payment.rovercode.com" >> $GITHUB_ENV
+          echo "STRIPE_SHARABLE_KEY=pk_live_auXa1Eh5nuIDvs2dyGwdxgRZ00HUdYWemF" >> $GITHUB_ENV
+
+      - name: Get branch name
```

Comment:
Does this store the branch name somewhere that's used elsewhere?

Reply:
It is used in the bucket name here: https://github.com/rovercode/rovercode-ui/blob/github-actions/.github/workflows/deploy.yml#L61